src/group_theory/abelianization.lean

```diff
@@ -63,6 +63,9 @@ def of : G →* abelianization G :=
   map_one' := rfl,
   map_mul' := λ x y, rfl }
 
+@[simp] lemma mk_of {a : G} :
```

Comment:
The name (`abelianization.mk_of`) is just a guess, happy to change if desired.

Reply:
`mk_eq_of` is probably better